thermo_nerf/scripts/train_eval_script.py

```diff
@@ -47,6 +48,8 @@ class TrainingParameters:
     """Flag to use settings for cold temperatures"""
     camera_optimizer_mode: Literal["off", "SO3xR3", "SE3"] = "SO3xR3"
     """Pose optimization strategy to use. Recommended to be SO3xR3."""
+    use_thermal: bool = True
```

Comment:
I'm not a fan of using booleans for the API because I think it leads to confusion in the long run. E.g. here `use_thermal` is only valid for nerfacto right? I think it would be better to instead create a new `ModelType` (called `ThermalNerfacto` and do the data change only in that case